demo/site/src/middleware.ts

```diff
@@ -28,9 +28,10 @@ export const config = {
          * - _next/image (image optimization files)
          * - favicon.ico, favicon.svg, favicon.png
          * - manifest.json
+         * - assets (assets from /public folder)
          * - robots.txt
          */
-        "/((?!api|_next/static|_next/image|favicon.ico|favicon.svg|favicon.png|manifest.json|robots.txt).*)",
+        "/((?!api|_next/static|_next/image|favicon.ico|icon.svg|apple-icon.png|manifest.json|assets|robots.txt).*)",
```

Comment:
```suggestion
        "/((?!api|_next/static|_next/image|favicon.ico|icon.svg|apple-icon.png|manifest.json|assets/|robots.txt).*)",
```

else you break any page name starting with assets

Reply:
We haven't considered it then. But we can fix it now 😉

Reply:
I am open to suggestions. 😄 How do you want to get it?

But right now, the loading of all public next resources is broken - no icons can be loaded,...

Furthermore, I think it's not a real problem in the demo because we have language-scoped URLs and no language called `assets`.

<img width="2553" alt="Screenshot 2025-01-14 at 07 47 01" src="https://github.com/user-attachments/assets/053f025a-6c0d-4d28-bdb2-28df228560f2" />

Reply:
> I am open to suggestions. 😄 How do you want to get it?

I'd just implement @nsams' suggestion.

> Furthermore, I think it's not a real problem in the demo because we have language-scoped URLs and no language called assets.

Yes, for Demo this will work. However, if a project doesn't use language scoped URLs, it won't work. And since Starter and Demo should be the same, we should change it in both.

Reply:
updated to suggestions.